Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -509,6 +517,18 @@ extension TelemetryWrapper {
             GleanMetrics.Widget.mTopSitesWidget.add()
         case (.action, .tap, .pocketStory, _, _):
             GleanMetrics.Pocket.openStory.add()
+        // Library
+        case (.action, .view, .libraryPanel, let type, _):
```

Comment:
Should the method here be `.tap` instead of .view?

Reply:
fixed in cleanup commits